code/datums/instability_meltdown.dm

```diff
@@ -0,0 +1,175 @@
+/datum/instability_meltdown
+	/// How likely a meltdown is to be picked
+	var/meltdown_weight = 0
```

Comment:
If almost all of these are `1` why didn't you default it to 1?

Reply:
I am not 100% sure what adding something with a weight of 0 does but I don't think it actually makes it _totally_ impossible for it to roll, unintuitively.
If this is to support the abstract fatal type, you might need to add an abstractness check as well (`var/abstract_type = typepath` then check against that path is a pattern we often use, so it doesn't need to be overridden in all children)